docs/components/Combobox/Web.stories.tsx

```diff
@@ -12,52 +13,61 @@ export default {
 } as ComponentMeta<typeof Combobox>;
 
 const ComboboxButton: ComponentStory<typeof Combobox> = args => {
+  const [selected, setSelected] = useState(1);
+
   return (
-    <Combobox {...args}>
-      <Combobox.TriggerButton
-        label="Select a teammate"
-        variation="subtle"
+    <>
+      <ClearButton
+        label="Clear Selection"
         type="primary"
-        icon="arrowDown"
-        iconOnRight={true}
+        onClick={() => setSelected(null)}
```

Comment:
typescript isn't happy with this. we'll wanna type the `selected` state as 

`useState<number | null>` and then this will stop complaining